pkg/cmd/copytable/copytable.go

```diff
@@ -0,0 +1,34 @@
+package copytable
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	cmdName = "copy-table"
+
+	shortDescription = "Copies dynamoDB records from a source to a target table"
+)
+
+func New(config *viper.Viper) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:       fmt.Sprintf("%s <source-table> <target-table>", cmdName),
+		Short:     shortDescription,
+		Args: cobra.ExactArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+		},
+	}
+
+	cmd.Flags().StringP("source-region", "s", "eu-west-1", "Set the AWS region for the source table")
```

Comment:
In the future, default perhaps could be read from env variable `AWS_DEFAULT_REGION`

Reply:
My idea is for this cmd to be a wrapper around the sdk. So it should use the sdk's env vars.